orchestra/views.py

```diff
@@ -187,3 +193,49 @@ def submit_task_assignment(request):
 # A simple status endpoint for things like health checks, etc.
 def status(request):
     return HttpResponse('OK')
+
+
+class OrchestraRegistrationView(RegistrationView):
```

Comment:
I wanted to, but the parent class emits the signal at the end of it. Would this be a problem if another project was using registration package and listening to the signal but didn't want to get notified by orchestra users?